src/Components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,114 @@
+
+import React, {useState,useEffect} from "react";
+
+import logo from '../../Images/joecoffee_logo-262x68.png';
```

Comment:
Please, rename images names. Make them more understandable.
For example : search-icon.png